docs/DeveloperGuide.adoc

```diff
@@ -62,7 +71,6 @@ For example, the `Logic` component (see the class diagram given below) defines i
 image::LogicClassDiagram.png[]
```

Comment:
![Screenshot 2019-10-25 at 1 18 27 PM](https://user-images.githubusercontent.com/42567064/67545026-30beab80-f72a-11e9-962a-8f49949204a3.png)

Words are too small -  not unreadable
May be cluttered with too many unnecessary details
Should give explanation of how it works